src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -1027,6 +1027,9 @@ namespace HVACVariableRefrigerantFlow {
 			VRFRTF = min( 1.0, ( CyclingRatio / PartLoadFraction ) );
 
 			VRF( VRFCond ).ElecHeatingPower = ( VRF( VRFCond ).RatedHeatingPower * TotHeatCapTempModFac ) * TotHeatEIRTempModFac * EIRFPLRModFac * VRFRTF * InputPowerMultiplier;
+
+			// adjust defrost power based on heating RTF
+			VRF( VRFCond ).DefrostPower *= VRFRTF;
```

Comment:
The problem here at this later point is that apply VRFRTF at this point as a correction means that it is not included in the LoadDueToDefrost term above and so the HeatingPLR is not accurate.  

				HeatingPLR += LoadDueToDefrost / TotalCondHeatingCapacity;

It is circular because you don't know the RTF until later, so I think it may need to iterate, or use a lagged RTF or something.